x-pack/plugins/security_solution/public/resolver/store/data/selectors.test.ts

```diff
@@ -417,7 +433,7 @@ describe('data state', () => {
         payload: {
           result: tree,
           // this value doesn't matter
-          databaseDocumentID: '',
+          parameters: mockTreeFetcherParameters(),
```

Comment:
Many tests need to specify the parameters even when the value isn't relevant to the test.